core/src/androidMain/kotlin/Observers.kt

```diff
@@ -52,7 +52,13 @@ internal class Observers(
             }
         } finally {
             if (observers.decrementAndGet(characteristic) < 1) {
-                peripheral.stopNotifications(characteristic)
+                try {
+                    peripheral.stopNotifications(characteristic)
+                } catch (e: NotReadyException) {
+                    // Silently ignore as it is assumed that failure is due to connection drop, in which case Android
+                    // will clear the notifications.
+                    println("Stop notification failure ignored.")
```

Comment:
Do you have a logger available instead of println?

Reply:
Tracking via #6.
https://github.com/JuulLabs/tuulbox/pull/22 might provide the logging capabilities we're looking for. 😉